recipes/pymdown-extensions/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pymdown-extensions" %}
+{% set version = "6.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6cf0cf36b5a03b291ace22dc2f320f4789ce56fbdb6635a3be5fadbf5d7694dd
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - markdown >=3.0.1
+    - pip
+    - python
+  run:
+    - markdown >=3.0.1
+    - python
+
+test:
+  imports:
+    - pymdownx
+    - tests.test_extensions
```

Comment:
Should this package be included in the install?

Reply:
took out `test.test_extensions` in the import tests